### ISSUE TITLE
dosent compile it says SD2Card does  not name a type 

### ISSUE THREAD

**Issue by brianeric26** [whizzosoftware/SIO2Arduino#15]
I included the latest sdfat library and it say this error , tried changing the line of sd2card to sdcard and then says another error
tried using 6 different realeses of arduiono ide they all gave me different errors 
finally i just tried latest ide realese then an early version of the sdfat library and still it could compile 
i barely undertand how to program an arduino so im very lost here  really rrally frustated that i build the board and cant even get then file to compile

**Comment by jcook793** [comment]
I ran into the same issue when compiling the sketch. I looked through the closed issues and saw that you need an older version of SdFat, this issue mentioned 1.1.4: https://github.com/whizzosoftware/SIO2Arduino/issues/9#issuecomment-696878623

You may have to remove "SdFat 2" from your libraries. After install SdFat 1.1.4, I did, as the IDE kept complaining about it.

**Comment by brianeric26** [comment]
Wow haha i has passed a long time since i asked this but yeah thay was what
i did , still i never got my atari to recognize the device

**Comment by jcook793** [comment]
Same here, actually. I spent a long time on it yesterday. I used some debug logging to figure out the problem happens when the [root directory is accessed](https://github.com/whizzosoftware/SIO2Arduino/blob/master/SIO2Arduino.ino#L101). I don't know enough about the SdFat library to grok what's going wrong. But I was able to successfully run SdFat's [`DirectoryFunctions`](https://github.com/greiman/SdFat/blob/1.1.4/examples/DirectoryFunctions/DirectoryFunctions.ino) example sketch. I'm thinking now there must be some core Arduino library that has changed with newer versions of the IDE.
